src/Aquifer.API/Endpoints/Projects/ProjectResourceStatusCounts.cs

```diff
@@ -25,20 +25,21 @@ public class ProjectResourceStatusCounts
         ResourceContentStatus.TranslationNotApplicable
     ];
 
-    internal static readonly List<ResourceContentStatus> NotStartedStatuses =
+    internal static readonly List<ResourceContentStatus> AwaitingAiDraftStatuses =
```

Comment:
I'm almost wondering if these should still be called `NotStartedStatuses` and we revert those renames on the frontend as well. The main reason being `AiDraftComplete` and `New` (i.e. Aquiferize not started) aren't really "awaiting AI draft". Thinking about the commonality between the three statuses it feels like they're all considered Not Started from the project's perspective. Thoughts?

Reply:
I think that's fair. The next step is EditorReview, so we can be saying that editor review hasn't started, which from a project's point of view, is the relevant bit.

Reply:
I can see that for the purpose of project count reporting. there were a lot of changes that had to be merged from master and a few things like this were just easier to keep as-is to make sure nothing broke.